persist-cli/src/main/resources/cli-help/ballerina-persist-generate.help

```diff
@@ -14,5 +14,5 @@ EXAMPLES
           $ bal persist generate --help
 
        Generate client objects for the model definition file inside persist directory.
-          $ bal persist generate
+          $ bal persist generate --module persist --datastore mysql
```

Comment:
SYNOPSIS
                bal persist generate [--module <module name>]
                                        [--datastore <store type>]
                                        [<-h> | <--help>]

OPTIONS
       --module <module name>
              The name of the module to initialize the persistence layer. If not specified, the default module will be used.
       --datastore <store type>
              The type of the datastore to be used for persistence.
              The supported datastores are 'inmemory', 'mysql', 'mssql', 'postgresql', 'googlesheets', and redis.
       -h, --help
           Print the usage details of all commands.

Reply:
Fixed in 48cc647